components/Card.js

```diff
@@ -21,13 +21,29 @@ export default function Card({
         </DeleteButton>
       )}
       <p>Date: {date}</p>
-      {weight && <p>Weight: {weight}</p>}
-      {height && <p>Height: {height}</p>}
-      {feastTime && (
-        <p>
-          Nurse time: {feastTime.substr(0, 2)}:{feastTime.substr(2, 2)}:
-          {feastTime.substr(4, 2)}
-        </p>
+      {weight?.[0] && (
```

Comment:
the naming of your "weight" variable is confusing. Are that multiple items? If so please rename it to weightList or weights.

Reply:
done